sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -234,6 +243,14 @@ export { PagedAsyncIterableIterator }
 
 export { PageSettings }
 
+// @public
+export interface ParsedKeyVaultIdentifier {
+    collection: "keys" | "secrets" | "certificates";
```

Comment:
I wonder if we should expose the original ID as well, just because (easy to go back and forth). What do people think?

Reply:
I like it! How would you name the property? "id"?

Reply:
For you, yes. That would be idiomatic, so `Id` for .NET (and probably the only one to capitalize).

Reply:
I added an `id` property. We don't capitalize properties, generally speaking.